src/applications/mhv-secure-messaging/components/Draft/DeleteDraft.jsx

```diff
@@ -91,11 +92,11 @@ const DeleteDraft = props => {
       {/* TODO add GA event */}
       <button
         type="button"
-        id="delete-draft-button"
+        id={`delete-draft-button${draftSequence ? `-${draftSequence}` : ''}`}
         ref={deleteDraftButtonRef}
         className={`usa-button usa-button-${
           cannotReply
-            ? 'primary vads-u-padding-x--4'
+            ? 'secondary vads-u-padding-x--4'

```

Comment:
`secondary` seems to be no longer conditional to `cannotReply`

`className={usa-button usa-button-secondary ${cannotReply ? vads-u-padding-x--4 : vads-u-flex--1}`